v2/component_analyses.yaml

```diff
@@ -0,0 +1,197 @@
+openapi: 3.0.1
+info:
+  title: Analytics Core API Specs for Version 2
+  version: "2"
+servers:
+- url: https://f8a-analytics-2445582058137.production.gw.apicast.io/api/v2
+  description: "Dependency Analytics Server"
+security:
+- tokenAuth: []
+paths:
+  /component-analyses/{eco}/{name}/{ver}:
+    get:
+      tags:
+      - Scan Services
+      summary: Get security and other details about a software component and recommendations
+        to change to a more secured version
+      operationId: da_scanner.api_v2.component-analyses
+      parameters:
+      - name: eco
+        in: path
+        description: component ecosystem
+        required: true
+        schema:
+          type: string
+      - name: name
+        in: path
+        description: component name
+        required: true
+        schema:
+          type: string
+      - name: ver
+        in: path
+        description: component version
+        required: true
+        schema:
+          type: string
+      - name: uuid
+        in: header
+        description: User Identification uuid
+        required: true
+        schema:
+          type: string
+
+      responses:
+        200:
+          description: Detailed response for the requested application component
+          content:
+            application/json:
+              schema:
+                items:
+                  oneOf:
+                    - $ref: '#/components/schemas/RegisteredUser'
+                    - $ref: '#/components/schemas/FreeUser'
+                    - $ref: '#/components/schemas/NoVulnerabilities'
+              examples:
+                RegisteredUser:
+                  value:
+                    uuid: "b3a51290-79a3-11ea-bb1f-48f17fb770f5"
```

Comment:
I think in yesterday's call we had discussed that it should be header, even in case of response. Any specific reason why its added to response body?

Reply:
Ack!